themes/default/content/blog/next-level-iac-pulumi-automation-api/index.md

```diff
@@ -0,0 +1,357 @@
+---
+title: "Next-level IaC: Pulumi Automation API and API Economy"
+date: 2024-04-26
+meta_desc: Learn how to use Pulumi Automation API to build your own Service API and expose it to your organization.
+meta_image: meta.png
+
+authors:
+
+- engin-diri
+
+tags:
+
+- next-level-iac
+- automation-api
+- api-economy
+- cloudformation
+- terraform
+- hcl
+
+---
+
+When I am talking with our customers and users, there is one common theme that comes up: how can we make infrastructure as code (IaC) more accessible to our organization? This is a great question, and it is something that we have been thinking about a lot at Pulumi.
```

Comment:
I think we should focus a bit more on how this functionality of Pulumi is unique. There is abc need out there, other IaCs and solutions struggle to solve and need xyz workarounds. With Pulumi, we have this custom built in so you can do abc easily.